.github/scripts/write-binary-content-via-raw-data.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash -e
+
+# This script creates a large binary resource (8 MiB) and verifies that its binary content
+# can be read correctly via both JSON/XML wrappers and direct binary upload.
+
+BASE="http://localhost:8080/fhir"
+
+# 8 MiB of random data, base64 encoded (8388608 bytes), kept in memory
+DATA="$(openssl rand -base64 8388608 | tr -d '\n')"
+# Also keep a raw binary version in memory for direct upload
+RAW_DATA="$(echo "$DATA" | base64 -d)"
+
+echo "Testing direct binary upload and download..."
+
+# Create Binary resource via direct binary upload
+ID_VIA_BINARY=$(curl -s -H 'Content-Type: application/octet-stream' --data-binary <(echo -n "$RAW_DATA") "$BASE/Binary" | jq -r '.id')
+
+echo "Created Binary resource via direct binary upload with ID: $ID_VIA_BINARY"
+
+# Retrieve and verify via direct binary
+BASE64_ENCODED_BINARY_RESOURCE_VIA_DIRECT=$(curl -s -H 'Accept: application/octet-stream' "$BASE/Binary/$ID_VIA_BINARY" | base64 | tr -d '\n')
+
+if [ "$DATA" = "$BASE64_ENCODED_BINARY_RESOURCE_VIA_DIRECT" ]; then
+    echo "✅ Direct Binary: Successfully verified 8 MiB binary content integrity"
+else
+    echo "🆘 Direct Binary: Content verification failed"
+    exit 1
+fi
+
+# Verify sizes (all in memory)
+ORIGINAL_SIZE=$(echo -n "$RAW_DATA" | wc -c)
```

Comment:
I mean you already have compared the data directly, why do you like to compare the length afterwards?